tests/pytests/unit/modules/test_xml.py

```diff
@@ -0,0 +1,98 @@
+"""
+    Tests for xml module
+"""
+
+
+import os
+import tempfile
+
+import pytest
+
+from salt.modules import xml
+
+
+@pytest.fixture
+def XML_STRING():
```

Comment:
```suggestion
def XML_STRING():
```
Lowercase the fixture name